src/app/modules/profile/components/profile/profile.component.html

```diff
@@ -10,9 +10,12 @@
       <li routerLinkActive="active">
         <a routerLink="./statistics">{{ "PROFILE.STATISTICS" | translate }}</a>
       </li>
-      <li routerLinkActive="active">
-        <a routerLink="./edit">{{ "PROFILE.EDIT" | translate }}</a>
+      <li routerLinkActive="active" *ngIf="getUserRole$ | async as userRole">
+        <a routerLink="./edit" *ngIf="userRole === 'couch' || userRole === 'administrator' ? true : false">{{ "PROFILE.EDIT" | translate }}</a>
```

Comment:
It's better to have prop in component like 
`ableToEditRoles = ['couch', 'administrator']`
and then in html check `*ngIf="ableToEditRoles.includes(userRole)"`

Reply:
Ok, I've corrected